addon/components/au-time-picker.js

```diff
@@ -1,16 +1,52 @@
 import Component from '@glimmer/component';
 import { action } from '@ember/object';
-import { tracked } from '@glimmer/tracking';
 import { isPresent } from '@ember/utils';
+import { trackedReset } from 'tracked-toolbox';
+import { formatTimeDigit } from '../helpers/format-time-digit';
 
 export default class AuTimePickerComponent extends Component {
-  @tracked hourValue = this.args.hours || 12;
-  @tracked minuteValue = this.args.minutes || 0;
-  @tracked secondValue =
-    isPresent(this.args.showSeconds) && !this.args.showSeconds
-      ? 0
-      : this.args.seconds || 0; // ignore 'seconds' argument if 'showSeconds' is false
-  @tracked keyCodes = [8, 9, 13, 33, 34, 37, 39, 46];
+  @trackedReset({
+    memo() {
+      return this.validateTimeValue(this.args.hours, 'hourValue');
+    },
```

Comment:
is this custom memoize function needed? Isn't providing the path enough? It will call the validateTimeValue twice now every time an arg is changed, right?